src/app/app.module.ts

```diff
@@ -14,14 +14,14 @@ import { TrendingInfoComponent } from './detail-page/detail-content/trending-inf
 import {  SimilarReposComponent } from './detail-page/detail-content/similar-repos/similar-repos.component';
 import { SearchBarComponent } from './search-bar/search-bar.component';
 import { HttpClientModule } from '@angular/common/http';
-import { SimilarRepoComponent } from './detail-page/detail-content/similar-repos/similar-repo/similar-repo.component';
+import { RequestRepoService } from './shared/request-repo.service';
 import { DetailContentComponent } from './detail-page/detail-content/detail-content.component';
 import { RefreshComponent } from './refresh/refresh.component';
 import { PageNotFoundComponent } from './page-not-found/page-not-found.component';
 import { LanguageFilterComponent } from './search-result/language-filter/language-filter.component';
 import { LanguageFilterPipe } from './shared/pipes/language-filter.pipe';
 import { RepoDesciptionPipe } from './shared/pipes/repo-description.pipe';
-import { RequestRepoService } from './shared/request-repo.service';
+
```

Comment:
Our app.module is too messy, try to split into multiple module [Ref](https://malcoded.com/posts/angular-fundamentals-modules/)